app/views/manifests/show.json.rabl

```diff
@@ -0,0 +1,9 @@
+object false
+
+node(:name)        { Setting.site_title }
```

Comment:
There should be a `short_name` as well as a `name`; the `short_name` is used for icons on the homescreen (whereas the `name` is used in the splash screen). In our case these can just be the same probably.